docs/readmes/README.managed-mode.md

```diff
@@ -0,0 +1,39 @@
+# Pod Identity in Managed mode
```

Comment:
Let's link this documentation to https://github.com/Azure/aad-pod-identity/blob/master/docs/readmes/README.md#features